system-int-test/test/cmr/system_int_test/ingest/translation_test.clj

```diff
@@ -47,19 +47,28 @@
   <Visible>true</Visible>
   </Collection>")
 
+(defn- convert-to-sets
+  "Convert lists in the umm record to sets so order doesn't matter during comparison"
+  [record]
+  (-> record
```

Comment:
Nitpick: I don't think the threading macro is necessary given the specificity of this function, but I like the thought behind it.